src/link.js

```diff
@@ -1,24 +1,51 @@
 var React = require('react');
 
 module.exports = React.createClass({
-  render: function () {
+  contextTypes: {
+    controller: React.PropTypes.object
+  },
+
+  componentDidMount() {
+
+    if (typeof this.props.signal === 'string') {
+      var signalPath = this.props.signal.split('.');
+      var signalParent = this.context.controller.signals;
 
-    if (typeof this.props.signal !== 'function') {
-      throw new Error('Cerebral React - You have to pass a signal to the LINK component');
+      while(signalPath.length - 1) {
+        signalParent = signalParent[signalPath.shift()];
+      }
+      this.signal = signalParent[signalPath];
+    } else {
+      this.signal = this.props.signal;
     }
 
-    if (typeof this.props.signal.getUrl !== 'function') {
-      throw new Error('Cerebral React - The signal passed is not bound to a route');
+    if (typeof this.signal !== 'function') {
+      throw new Error('Cerebral React - You have to pass a signal to the Link component');
```

Comment:
Sorry to be picky, but I think we need it inside the while loop too. As any step in the path may be invalid :-)